src/docker/remote.rs

```diff
@@ -34,33 +34,50 @@ fn subcommand_or_exit(engine: &Engine, cmd: &str) -> Result<Command> {
     Ok(engine.subcommand(cmd))
 }
 
+pub fn posix_parent(path: &str) -> Option<&str> {
+    Path::new(path).parent()?.to_str()
+}
+
 impl<'a, 'b, 'c> ContainerDataVolume<'a, 'b, 'c> {
-    fn create_dir(&self, dir: &Path, msg_info: &mut MessageInfo) -> Result<ExitStatus> {
+    // `path` should be a relative POSIX path to the root directory
+    // on windows, this should be something like `mnt/c`.
+    // that is, all paths inside the container should not have
+    // the mount prefix.
+
+    fn create_dir(
+        &self,
+        reldir: &str,
+        mount_prefix: &str,
+        msg_info: &mut MessageInfo,
+    ) -> Result<ExitStatus> {
```

Comment:
there is no argument called `path`, this should also probably be a `# Notes` in a doc-comment

Reply:
Does this look better? I've made them just regular comments since none of the private methods are doc comments, but I can convert them if desired.